docs/Oracle Jobs/Job Types/direct_request.md

```diff
@@ -166,4 +168,4 @@ observationSource   = """
 """
 ```
 
-Multi-word response jobs like this, and also large response jobs need to use the [Operator contract](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.7/Operator.sol) as the on-chain contract that coordinates things between the consuming contract and the Chainlink node, as opposed to the legacy [Oracle contract](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.6/Oracle.sol).
\ No newline at end of file
+Multi-word response jobs like this, and also large response jobs need to use the [Operator contract](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.7/Operator.sol) as the on-chain contract that coordinates things between the consuming contract and the Chainlink node, as opposed to the legacy [Oracle contract](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.6/Oracle.sol).
```

Comment:
I didn't touch this - maybe just CR-LF :shrug:

Reply:
The previous file had no newline at the end, so your editor probably automatically added it. My editor, Atom, does the same. We can ignore this.